homeassistant/const.py

```diff
@@ -474,7 +474,8 @@
 FREQUENCY_HERTZ: Final = "Hz"
 FREQUENCY_MEGAHERTZ: Final = "MHz"
 FREQUENCY_GIGAHERTZ: Final = "GHz"
-
+FREQUENCY_RPM: Final = "rpm"
```

Comment:
We don't add common units unless the unit is used in many integrations. Where is this unit needed?

Reply:
In the ocpp integration (custom). But I can imagine there are many others that use rpm for fan speeds.

Reply:
Please specifically list the integrations in the core that can use this. We won't accept the PR unless there's a need in core.

Reply:
> We don't add common units unless the unit is used in many integrations. Where is this unit needed?

I use it for my already submitted QNAP QSW integration, but there have been no reviews yet :)
https://github.com/home-assistant/core/pull/58053/files#diff-3064fa0e716a00d1f14cff440147a93a99a5d34bded9420f757b9e5deac485ecR44

Reply:
I'd like a complete list, thanks.

Reply:
@MartinHjelmare there you go:
https://github.com/home-assistant/core/blob/dev/homeassistant/components/comfoconnect/sensor.py#L161
https://github.com/home-assistant/core/blob/dev/homeassistant/components/comfoconnect/sensor.py#L177
https://github.com/home-assistant/core/blob/dev/homeassistant/components/smarty/__init__.py#L34
https://github.com/home-assistant/core/blob/dev/homeassistant/components/xiaomi_miio/number.py#L104
https://github.com/home-assistant/core/blob/dev/homeassistant/components/xiaomi_miio/number.py#L178
https://github.com/home-assistant/core/blob/dev/homeassistant/components/xiaomi_miio/sensor.py#L184
https://github.com/home-assistant/core/blob/dev/homeassistant/components/xiaomi_miio/sensor.py#L192
https://github.com/home-assistant/core/blob/dev/homeassistant/components/xiaomi_miio/sensor.py#L200

Reply:
Thanks! With three current integrations that could use the unit I think we can hold off a bit on adding this unit.

Reply:
> Thanks! With three current integrations that could use the unit I think we can hold off a bit on adding this unit.

I disagree on this since **rpm** is needed for any integration which exposes fan speeds and it looks like a pretty common case for me (personal opinion).
I think we could add this now so that new integration can use it from now on...

Reply:
We have 30 fan integrations already. If only 3 need it so far I don't think the need is enough yet. We can add this unit when there's a greater need.